css/yoga.css

```diff
@@ -219,6 +219,7 @@ header {
   color: green;
   font-weight: bolder;
   border: 2px solid green;
+  transition: 0.3s ease-in-out;
```

Comment:
If you want to code like a professional, there is no need like this:
```diff
- transition: 0.3s ease-in-out;
+ transition: .3s ease-in-out;
```

Overall looking Great.

Reply:
Sure Ali, I'll keep that in mind